astropy/utils/data.py

```diff
@@ -1124,47 +1153,61 @@ def is_url_in_cache(url_key):
 
     Returns
     -------
-    in_cache : bool
```

Comment:
You need to give the docstring `varname : vartype` combo. Removing it would cause malformatting of the docstring, so please add it back. Doesn't matter if you don't really do `return in_cache`.

Reply:
I thought I had tried this and `napoleon` did fine with no varname in return strings? I'll put it back.

Reply:
Just to clarify, it wouldn't error, but rather, I think, it formats `bool` like the variable name rather than the type. Not end of the world, but I like things to be consistent. 😅